sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerModelFactory.cs

```diff
@@ -172,6 +172,15 @@ public static FieldData FieldData(FieldBoundingBox boundingBox, int pageNumber,
             return new FieldData(boundingBox, pageNumber, text, fieldElements);
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="FieldValue"/> structure.
+        /// </summary>
+        /// <param name="type">The field type.</param>
+        /// <param name="text">The text detected related to the field.</param>
+        /// <returns>A new <see cref="FieldValue"/> instance for mocking.</returns>
+        public static FieldValue FieldValue(FieldValueType type, string text) =>
+            new FieldValue(type, text);
```

Comment:
I don't understand why we need to support this scenario in our factory. From the user's point of view, there's no `Text` property associated with `FieldValue`. In which mock scenario would users call this method?

Reply:
hmmmm .... great point... Idk... let me update

Reply:
ok. I went with internal testing instead of exposing it in the mocking class because you are right. This is not a scenario that is intended for the user to test.
Thank you :)